test/integration/sagemaker/test_distributed_operations.py

```diff
@@ -35,6 +36,29 @@ def test_dist_operations_multi_gpu(sagemaker_session, ecr_image, dist_gpu_backen
     _test_dist_operations(sagemaker_session, ecr_image, instance_type, dist_gpu_backend, 1)
 
 
+@pytest.mark.skip_cpu
+def test_dist_operations_fastai_gpu(sagemaker_session, ecr_image, py_version):
+    if py_version != 'py3':
+        print('Skipping the test because fastai supports >= Python 3.6.')
+        return
+
+    instance_type = 'ml.p3.8xlarge'
+    with timeout(minutes=8):
+        pytorch = PytorchTestEstimator(entry_point='train_cifar.py',
+                                       source_dir=os.path.join(fastai_path, 'cifar'),
```

Comment:
Do we have to keep the binary data files in the repo?

Reply:
Yes, if we want to use sagemaker input channels :(